operations/firmware/Updating_Firmware_without_FAS.md

```diff
@@ -0,0 +1,119 @@
+# Updating BMC Firmware and BIOS for NCNs without FAS
+
+**NOTE: This procedure should only be used if FAS is not available, such as during initial CSM install.**
+
+> **NOTE:**
+> On HPE nodes, the BMC Firmware is iLO 5 and BIOS is System ROM.
+> The commands in the procedure must be run on `ncn-m001`.
+
+## Prerequisites
+
+The following information is needed:
+
+* IP address of each NCN BMC
+* IP address of `ncn-m001`
+* `root` user password for each NCN BMC
+
+## Obtain the required firmware
+
+The firmware or BIOS can be obtained from the `HFP` tarball if it has been installed, or from the HPE Morpheus Server.

```

Comment:
Is Morpheus a customer-facing name? I was under the impression customers will be using the HPE Support Center (HPESC), which gets content from Morpheus.

Reply:
> Is Morpheus a customer-facing name? I was under the impression customers will be using the HPE Support Center (HPESC), which gets content from Morpheus.

I applied your linting suggestions, because they don't seem controversial, but I don't know the answer to this question. @mbuchmann-hpe will need to answer this one.

Reply:
Updated